packages/dds/matrix/src/matrix.ts

```diff
@@ -521,20 +523,34 @@ export class SharedMatrix<T = any>
 			SnapshotPath.cols,
 			this.cols.summarize(this.runtime, this.handle, serializer),
 		);
+		const cellsSnapshot = this.cells.snapshot();
+		const props: Record<string, number> = {
+			cellsSnapshotSize: cellsSnapshot.length,
+			rowCount: this.rowCount,
+			colCount: this.colCount,
+			setCellsCountFromLastSummary: this.setCellsCountFromLastSummary,
```

Comment:
I'm not sure how much interesting is setCellsCountFromLastSummary. It's not just a function of user actions, but also a function of summary frequency. I have hard time imagining how we will use it in telemetry. Consider removing, unless you have good case for it.

Reply:
Yeah, I was dicey too whether to add this or not. Removing it.